debian/incus.preinst

```diff
@@ -10,6 +10,38 @@ case "$1" in
         if ! getent group incus-admin >/dev/null; then
             addgroup --system incus-admin
         fi
+
+        if ! getent passwd incus >/dev/null; then
+            adduser --system --shell /usr/sbin/nologin --home /var/lib/incus --ingroup incus incus
```

Comment:
We'll want `--no-create-home` too I think.

Reply:
I agree, it is done like this with LXD snap package, but I don't see much value in it, I will change it.